library/x509_crt.c

```diff
@@ -662,7 +658,7 @@ static int x509_get_subject_alt_name( unsigned char **p,
         }
 
         /*
-         * Check that the SAN are structured correct.
```

Comment:
This change is outside of the PRs scope. I'd suggest not mixing in extra changes with the main ones. But this is far from being a blocker.